images/ose-insights-runtime-exporter.yml

```diff
@@ -0,0 +1,30 @@
+content:
+  source:
+    dockerfile: ./Containerfile-exporter
+    git:
+      branch:
+        target: release-{MAJOR}.{MINOR}
+      url: git@github.com:openshift-priv/insights-runtime-extractor.git
+      web: https://github.com/openshift/insights-runtime-extractor
+    ci_alignment:
+      streams_prs:
+        ci_build_root:
+          stream: rhel-9-golang-ci-build-root
+distgit:
+  branch: rhaos-{MAJOR}.{MINOR}-rhel-9
+  component: ose-insights-runtime-exporter-container
+delivery:
+  repo_name: openshift4/ose-insights-runtime-exporter-rhel9
+enabled_repos:
```

Comment:
Build log does not show any `dnf` things. Please remove the enabled_repos.

Reply:
fixed in the latest commit